src/assets/templates/widgets/gear_popover.html

```diff
@@ -26,20 +28,19 @@
 
   <div class="cc-popover cc-dropdown cc-launcher-dropdown-menu" data-ng-show="api.popover.status('cc-popover-menu')">
     <div>
-      <div class="cc-launcher-status-loading"
-        data-cc-loading-directive="statusLoading"
-        data-cc-loading-process="Loading..."
-        data-ng-if="statusLoading">
+      <div class="cc-launcher-status-loading" data-cc-loading-directive="statusLoading"
+        data-cc-loading-process="Loading..." data-ng-if="statusLoading">
       </div>
       <div class="cc-popover-title">
         <span><strong>Student ID: </strong><span data-ng-bind="api.user.profile.campusSolutionsID"></span><span>
       </div>
-      <div data-ng-if="!statusLoading && !api.user.profile.delegateActingAsUid && (count !== 0 || hasBillingData || regStatus.hasData)">
-        <div class="cc-popover-noitems" data-ng-if="count === 0">
+      <div
+        data-ng-if="!statusLoading && !api.user.profile.delegateActingAsUid && (badgeCount !== 0 || hasBillingData || regStatus.hasData)">
+        <div class="cc-popover-noitems" data-ng-if="badgeCount === 0">
           <i class="cc-left fa fa-check-circle cc-icon-green"></i>
           <strong>No Active Alerts</strong>
         </div>
-        <div data-ng-include="'widgets/status_popover.html'" data-ng-if="count !== 0"></div>
+        <div data-ng-include="'widgets/status_popover.html'" data-ng-if="badgeCount !== 0"></div>
```

Comment:
@nkutub  is this the only "non-formatting" related change to the file? `badgeCount` vs `count`?

Reply:
yes all the counts turned into badgeCount